### DISCUSSION TITLE
CamelCase for all the output fields

### DISCUSSION THREAD

**Discussion by exactstat** [fastapi-users/fastapi-users#969] [Ideas]
Hi!
I know it's possible to make a camel case for user models by adding the code to the Pydantic Model:
```
class Config:
    alias_generator = to_camel
    allow_population_by_field_name = True
```

**THE QUESTION: how to make a camel case in the BearerResponse model?**

I've tried the solution below, but it 
1) allows me to login at the auth/jwt/login endpoint
2) gives me an unauthorized error on any endpoint with authorization required

**The incorrect solution I came up with:**

```
import fastapi_users.authentication
from fastapi_users.authentication import BearerTransport
from pydantic import BaseModel
import humps


class BearerResponse(BaseModel):
    access_token: str
    token_type: str

    class Config:
        alias_generator = humps.camelize
        allow_population_by_field_name = True

# Here the origin model is replaced by the modified one (for a camel-case).
fastapi_users.authentication.transport.bearer.BearerResponse = BearerResponse

bearer_transport = BearerTransport(tokenUrl="auth/jwt/login")
```

**Comment by frankie567** [comment]
Hi @exactstat!

When you say:

> gives me an unauthorized error on any endpoint with authorization required

Do you try from the Swagger/OpenAPI interactive documentation? If so, it's expected: the interactive doc *expects* the BearerResponse to be structured with `access_token` and `token_type` (in snake case) to be able to automatically retrieve the token.

Other than that, I don't see anything wrong with what you did.

**Reply by exactstat** [reply]
Yes, it's interactive docs. Swagger. It's an interesting situation :) The fix works, but I lose the documentation functionality after that. Ok, thank you, I will try to cope with that.
